<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_chat</class>
 <widget class="QDialog" name="Dialog_chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
border-image: url(:/pictures/img/background_dialog.png);
}</string>
  </property>
  <widget class="QPushButton" name="validate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/pictures/img/validate.png);
}

QPushButton::hover {
	border-image: url(:/pictures/img/validate_hover.png);

}

QPushButton::pressed {
	border-image: url(:/pictures/img/validate_pressed.png);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>31</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>999999</width>
     <height>999999</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
border-image : &quot;&quot;;

font: 16pt &quot;Futura Lt BT&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/pictures/img/cancel.png);
}

QPushButton::hover {
	border-image: url(:/pictures/img/cancel_hover.png);

}

QPushButton::pressed {
	border-image: url(:/pictures/img/cancel_pressed.png);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>579</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(135, 97, 79);
padding: (-50,-50,-50,-50);
border-image:&quot;&quot;;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Lt BT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
  background-color: #00000000;
  padding-top: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-bottom: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-left: 4px;
  padding-right: 4px;
  border-style: solid;
  border-left: 2px solid rgb(141, 121, 99);
  border-bottom: 2px solid rgb(141, 121, 99);
  border-radius: 2%;
}


QComboBox QAbstractItemView {
  background-color:  rgb(181, 163, 136);
  border-radius: 4px;
  border: 1px solid #32414B;
  selection-color: #148CD2;
  selection-background-color: rgb(255, 211, 179);
}


QComboBox::indicator {
  background-color:transparent;
  selection-background-color: transparent;
  color: transparent;
  selection-color: transparent;
  /* Needed to remove indicator - fix #132 */
}

QComboBox::indicator:alternate {
  background: #19232D;
}

QComboBox::item:alternate {
  background: #19232D;
}

QComboBox::item:checked {
  font-weight: bold;
}

QComboBox::item:selected {
  border: 0px solid transparent;
}

QComboBox::drop-down {
  subcontrol-origin: padding;
  subcontrol-position: top right;
  width: 20px;
  border-left-width: 0px;
  border-left-color: #32414B;
  border-left-style: solid;
  border-top-right-radius: 3px;
  border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
  image:url(:/pictures/img/down_arrow.png);
height: 20%;
width: 20%;
}

QComboBox::down-arrow:on, QComboBox::down-arrow:hover, QComboBox::down-arrow:focus {
  image: url(:/pictures/img/down_arrow_hover.png);
height: 20%;
width: 20%;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Lt BT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  background-color: #00000000;
  padding-top: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-bottom: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-left: 4px;
  padding-right: 4px;
  border-style: solid;
  border-left: 2px solid rgb(141, 121, 99);
  border-bottom: 2px solid rgb(141, 121, 99);
  border-radius: 2%;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Titre</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
